geps/gep-1619.md

```diff
@@ -163,43 +163,258 @@ Java application servers such as Tomcat and Jetty, were the first to standardize
 
 ### Session Persistence API
```

Comment:
Can we say something about the applicability of this API to [GAMMA](https://gateway-api.sigs.k8s.io/contributing/gamma/) since session persistence in a mesh is probably a very common use-case?

Reply:
I think all Gateway APIs and policies do apply by default to mesh - unless explicitly excluded ( and I don't think we have any case yet ?). Even the Gateway CR is used in Istio mesh for waypoints in ambient.

But it's good to have a section - in particular around the security/privacy differences between the 2 cases.

Reply:
@costinm for (m)TLS/certs there are proposals in the works that are for the Gateway/Ingress use-case and the mesh use-cases are specifically excluded e.g. you can see this comment https://github.com/kubernetes-sigs/gateway-api/pull/2113#discussion_r1231605636 and it seems you also agree with that comment. So it is good to call out whether a given proposal applies to mesh (or not).